docs/website/docs/command-reference/delete.md

```diff
@@ -0,0 +1,38 @@
+---
+title: odo delete
+sidebar_position: 2
+---
+
+### Deleting a component
```

Comment:
Could you start with a level-2 paragraph (`##`) instead? The level-1 is inserted automatically by docusaurus, and we should use the following one.

If we want titles to be displayed in smaller fonts, we should act on style instead.